pkg/proxy/backup.go

```diff
@@ -228,7 +228,19 @@ func (p *Proxy) BackupGet(ctx context.Context, req *rpc.EngineBackupGetRequest)
 		Backup: &rpc.EngineBackupInfo{},
 	}
```

Comment:
NIT: same here. don't need to init the response at the beginning.

Reply:
Updated.